scripts/yarn-checks-ci.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+source "${0%/*}/helpers.sh"
```

Comment:
Can we use `./helpers.sh` or `./scripts/helpers.sh` here?

Reply:
Used `./scripts/helpers.sh`.